src/screens/SendAmountInput.js

```diff
@@ -133,22 +138,17 @@ class SendAmountInput extends React.Component {
                 flexDirection: 'row', alignItems: 'center', justifyContent: 'space-between', marginTop: 40,
               }}
               >
-                <View style={{ width: 80, height: 40 }} />
+                {renderGhostElement()}
                 <AmountTextInput
                   ref={this.inputRef}
                   autoFocus
                   onAmountUpdate={this.onAmountChange}
                   value={this.state.amount}
                 />
-                <View style={{
```

Comment:
What about this view?

Reply:
Really? Because it had many styles in it. But, if it was not necessary, you can mark it as resolved.

Reply:
I imagine it was something we had before we refactored the element into `TokenBox`.